Core/AppDeepLinks.swift

```diff
@@ -37,12 +37,15 @@ public struct AppDeepLinks {
     
     public static func isQuickLink(url: URL) -> Bool {
         if let scheme = url.scheme {
-            return AppDeepLinks.quickLink.contains(scheme)
+            return AppDeepLinks.quickLink.lowercased().contains(scheme.lowercased())
         }
         return false
     }
     
     public static func query(fromQuickLink url: URL) -> String {
-        return url.absoluteString.replacingOccurrences(of: quickLink, with: "")
+        let components = NSURLComponents.init(url: url, resolvingAgainstBaseURL: true)!
```

Comment:
The convention here is not to use `.init(` ... just use `NSURLComponents(` directly.

Reply:
I believe this is now resolved in https://github.com/duckduckgo/iOS/pull/274/commits/695e8ca0380dd1872fdc360f2cd519f2886b2931.